site/content/api/Command-And-Control-API.md

```diff
@@ -12,43 +12,39 @@ THIS IS A FIRST DRAFT VERSION OF THE API SPEC AND THUS STILL SUBJECT TO DISCUSSI
 IF YOU THINK THAT THE SPEC SHOULD BE CHANGED OR CAN BE IMPROVED PLEASE DO NOT HESITATE TO OPEN AN ISSUE OR DROP US A LINE ON THE MAILING LIST.
 {{% /warning %}}
 
-A client can be a device connected to Hono directly (using an AMQP connection) or via a *Protocol Adapter* (for example via MQTT,
-LWM2M, etc). Devices and *Protocol Adapters* can also poll Hono periodically in order check for new command & control messages without
-maintaining a constant connection to Hono infrastructure.
+The Command & Control API is defined by means of AMQP 1.0 message exchanges, i.e. a client needs to connect to Hono using AMQP 1.0 in order to invoke operations of the API as described in the following sections. Throughout the remainder of this page we will simply use AMQP when referring to AMQP 1.0.
 
-The Command & Control API is defined by means of AMQP 1.0 message exchanges, i.e. a client needs to connect to Hono using an AMQP 1.0 client in 
-order to invoke operations of the API as described in the following sections.
+# Operations
 
-# Southbound Operations
+The following API defines operations that can be used by *Business Applications* to send control messages to *Protocol Adapters* (and by *Protocol Adapters* to receive them) as well as the response from the *Protocol Adapters* back to the *Business Applications*.
 
-The following operations can be used by *Devices* and/or *Protocol Adapters* to receive command & control messages sent by *Business Applications*.
-
-## Receive Control Message
+## Send and Receive a Command
 
 **Preconditions**
 
-1. Client has established an AMQP connection with Hono.
-2. Client has established an AMQP link in role *receiver* with Hono using target address `control/${tenant_id}/${device_id}`. Where `${device_id}` is the ID of the device to receive control messages for and `${tenant_id}` is the ID of the tenant the device belongs to.
+1. *Protocol Adapter* and *Business Application* have established an AMQP connection with the AMQP 1.0 Network.
+2. *Protocol Adapter* has established an AMQP link in role *receiver* with the AMQP 1.0 Network using target address `control/${tenant_id}/${device_id}`. Where `${device_id}` is the ID of the device to receive control messages for and `${tenant_id}` is the ID of the tenant the device belongs to. 
```

Comment:
Thats true for the AMQP links. For a typical reader I find it counter-intuitiv to call this source address, since it is the address to which the sender wants to send the message. But it is also confusing to call it target address, because that is wrong in terms of AMQP. Maybe I just write 'address'?

Reply:
Since the whole API is defined in terms of AMQP 1.0, I would really appreciate if we could use at least the correct terminology. BTW we are already doing so in all other APIs.